features/bookings.feature

```diff
@@ -0,0 +1,41 @@
+Feature: Testing meeting room booking API
+  As a user
+  I want to book a meeting room
+
+  Background:
+    Given I am a registered user
+
+  Scenario: Booking an available meeting room
+    Given A meeting room is "available"
+    When I make a POST api request to endpoint "/v1/bookings" with below params:
+      | start_time | 2099-09-29 10:00:00 |
+      | end_time   | 2099-09-29 10:30:00 |
+    Then my booking is made successfully
+    And the response code is "201"
+
+  Scenario: Booking an unavailable meeting room
+    Given A meeting room is "unavailable"
+    When I make a POST api request to endpoint "/v1/bookings" with below params:
+      | start_time | 2099-09-29 10:00:00 |
+      | end_time   | 2099-09-29 10:30:00 |
```

Comment:
Shouldn't there be a meeting_id here?

Reply:
Yes @glen , As discussed earlier in the call. I will pass meeting_room_id here.